src/ICeWall/doc/set_core_area.md

```diff
@@ -0,0 +1,11 @@
+## ICeWall set_core_area
+### Synopsys
+```
+  % ICeWall set_core_area <minX> <minY> <maxX> <maxY>
+```
+### Description
+Specify the co-ordinates of the chip core. This area is reserved for stdcell placements. Placement of padcells should be made outside this core area, but within the specified die area
```

Comment:
Helpful to specify micrometers.

Reply:
Why not take the core area from the db?

Reply:
If you evisage a flow where you start with an existing die, and extract a footprint from that to be used for the next chip, then I think it's useful to have a way to carry that infomation in a footprint file - the fact that there is a TCL command here is to complement the ability to specify it in the footprint file.
I guess this can serve as a "check", rather than a "set"

Reply:
That sounds reasonable to me - have a die area in the footprint file and ensure that it matches the die area in the DB.